app/helpers/cookies_policy_helper.rb

```diff
@@ -5,4 +5,17 @@ def render_cookie_entry(cookie_name, cookie_desc, cookie_domain = nil)
                      cookie_desc: cookie_desc,
                      cookie_domain: cookie_domain }
   end
+
+  def matomo_iframe_src
+    url = Spree::Config.matomo_url
+    url += "/index.php?module=CoreAdminHome&action=optOut"
+    url += "&language=#{locale_language}"
+    url + "&backgroundColor=&fontColor=222222&fontSize=16px&fontFamily=%22Roboto%22%2C%20Arial%2C%20sans-serif"
+  end
+
+  # removes country from locale if needed
+  #   for example, both locales en and en_GB return language en
+  def locale_language
+    I18n.locale[0..1]
+  end
```

Comment:
I'm missing unit tests for these two methods

Reply:
thanks for reminding :+1: